recipe/meta.yaml

```diff
@@ -1,128 +1,82 @@
+    - filterpy >=1.4.5  # [py<311]
+    - gluonts >=0.9
+    - h5py  # [py<312]
+    - hmmlearn  # [py<311]
+    - holidays
+    - keras-self-attention  # [py<311]
+    - kotsu >=0.3.1
+    - matplotlib-base >=3.3.2
+    - mne
+    - numba >=0.53,<0.59
+    - pmdarima >1.8.1,<3.0.0  # [py<312]
+    - prophet >=1.1  # [py<312]
+    - pykalman-bardo >=0.9.7,<0.10
+    - pyod >=0.8  # [py<312]
+    - scikit-optimize
+    - seaborn >=0.11
+    - seasonal
+    - skpro >=2,<2.2.0
+    - statsforecast <1.7.0,>=0.5.2  # [py<312]
+    - statsmodels >=0.12.1
+    - stumpy >=1.5.1  # [py<311]
+    - tbats >=1.1  # [py<312]
+    - tensorflow  # [py<312]
+    - tsfresh >=0.17  # [py<312]
+    - tslearn >=0.5.2,<0.6.0  # [py<311]
+    - xarray
 
-outputs:
-  - name: sktime
-    build:
-      script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
-    requirements:
-      host:
-        - python
-        - pip
-        - setuptools
-        - wheel
-      run:
-        - python
-        - deprecated >=1.2.13
-        - numba >=0.53
-        - numpy >=1.21.0,<1.25
-        - pandas >=1.1.0,<2.0.0
-        - scikit-learn >=0.24.0,<1.3.0
-        - scipy >=1.2.0,<2.0.0
-      run_constrained:
-        - esig ==0.9.7
-        - filterpy >=1.4.5
-        - gluonts >=0.9.0
-        - hmmlearn >=0.2.7
-        - kotsu >=0.3.1
-        - matplotlib-base >=3.3.2
-        - pmdarima >=1.8.0,!=1.8.1,<3.0.0
-        - prophet >=1.1
-        - pykalman >=0.9.5
-        - pyod >=0.8.0
-        - scikit-posthocs >=0.6.5
-        - seaborn >=0.11.0
-        - statsforecast >=0.5.2
-        - statsmodels >=0.12.1
-        - stumpy >=1.5.1
-        - tbats >=1.1.0
-        - tsfresh >=0.17.0
-        - tslearn >=0.5.2
-    test:
-      requires:
-        - pip
-      commands:
-        - pip check
-      imports:
-        - sktime
-        - sktime.classification
-        - sktime.datasets
-        - sktime.distances
-        - sktime.forecasting
-        - sktime.performance_metrics
-        - sktime.regression
-        - sktime.series_as_features
-        - sktime.transformations
-        - sktime.utils
-
-  - name: sktime-all-extras
-    build:
-      # Skip building this package because many of the dependencies are not available yet.
-      skip: true
-    requirements:
-      run:
-        - python
-        - {{ pin_subpackage("sktime", max_pin='x.x.x') }}
-        - attrs
-        - cloudpickle
-        - dask
-        - dtw-python
-        # - esig  # not on CF
-        - filterpy
-        - h5py
-        # - hmmlearn  # not on CF
-        - gluonts
-        - matplotlib-base
-        - mne
-        - numba >=0.53
-        - pmdarima
-        - prophet
-        - pykalman
-        - pyod
-        - scikit-posthocs
-        - seaborn
-        # - seasonal  # not on CF
-        - statsforecast
-        - stumpy
-        - tbats
-        - tsfresh
-        - tslearn
-        - xarray
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
can we run the tests?

Reply:
will do.

Reply:
The tests take at least 20 minutes on my laptop. There is something like 50k unit tests.

```
= 59685 passed, 2358 skipped, 7 xfailed, 4 xpassed, 382026 warnings in 1278.07s (0:21:18) =
```